.github/workflows/test.yml

```diff
@@ -1,8 +1,8 @@
+---
```

Comment:
can we remove this too? it seems like an artifact of the formatter you used at one point

Reply:
sure (this comes from [yaml specs](https://yaml.org/spec/1.2/spec.html#id2760395) but I have no strong opinion on whether it should be used or not. As far as I understand, this is not mandatory).